plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/util.ts

```diff
@@ -59,3 +59,42 @@ export const parseRepoUrl = (repoUrl: string) => {
 
   return { host, owner, repo, organization };
 };
+
+export const parseRepoUrlForBitbucket = (repoUrl: string) => {
```

Comment:
Seem like we'd be able to use the existing repo URL parsing function to this, but perhaps add fields to the result